loops.js

```diff
@@ -5,7 +5,11 @@
 // Hoera!
 // Hoera!
 // ==========================================
+const birthday = "Hoera!";
 
+for (let i = 0; i<3; i++) {
+    console.log(birthday)
```

Comment:
; aan het einde van de zin

Reply:
Helemaal waar, echt nog een aandachtspuntje voor me!
Enorm bedankt voor het nakijken in ieder geval!